src/main/java/io/jenkins/plugins/postgresql/DataConversionHandler.java

```diff
@@ -0,0 +1,197 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2020, Jenkins project contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package io.jenkins.plugins.postgresql;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.model.Fingerprint;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.*;
+
+public class DataConversionHandler {
+
+    private static String serialize(Fingerprint.RangeSet src) {
+        StringBuilder buf = new StringBuilder(src.getRanges().size() * 10);
+        for (Fingerprint.Range r : src.getRanges()) {
```

Comment:
Naming: r->range

Reply:
Explained in https://github.com/jenkinsci/postgresql-fingerprint-storage-plugin/pull/3#discussion_r463094984. Will keep a note of it and consider refactoring in Jenkins core in a separate PR.